kore/src/Kore/Repl/Parser.hs

```diff
@@ -251,7 +251,7 @@ claimIndexParser :: Parser ClaimIndex
 claimIndexParser = ClaimIndex <$$> Char.string "c" *> decimal
 
 ruleNameParser :: Parser RuleName
-ruleNameParser = RuleName <$$> quotedOrWordWithout ""
+ruleNameParser = RuleName <$$> quotedOrWordWithout "|>"
```

Comment:
So the problem was `|` parsed as part of the command and not as piping operator?

Reply:
That's right.